mapie/conformity_scores/conformity_scores.py

```diff
@@ -158,20 +183,151 @@ def get_conformity_scores(
 
         Parameters
         ----------
+        X: NDArray
+            Observed feature values.
+
         y: NDArray
-            Observed values.
+            Observed target values.
 
         y_pred: NDArray
-            Predicted values.
+            Predicted target values.
 
         Returns
         -------
         NDArray
             Conformity scores.
         """
-        conformity_scores = self.get_signed_conformity_scores(y, y_pred)
+        conformity_scores = self.get_signed_conformity_scores(X, y, y_pred)
         if self.consistency_check:
-            self.check_consistency(y, y_pred, conformity_scores)
+            self.check_consistency(X, y, y_pred, conformity_scores)
         if self.sym:
             conformity_scores = np.abs(conformity_scores)
         return conformity_scores
+
+    @staticmethod
+    def _get_quantile(
```

Comment:
UPD: you can remove the `_` in the beginning of the function.

Reply:
QUESTION: the functino `compute_quantiles` exists in `utils.py`. Is it possible to use it instead of defining a new method that does the same thing?

Reply:
With the function in `ùtils.py`` it is not possible to provide the axis, method or alpha parameters.